docs/rules/strict.md

```diff
@@ -139,6 +141,22 @@ function foo() {
 }());
 ```
 
+```js
```

Comment:
Excellent new example!

Because the style rule for red background color supports only up to 3 code fences, suggest merging the second and thirds code fences above as follows:

```js
/*eslint strict: ["error", "function"]*/

function foo() {
}

(function() {
    function bar() {
        "use strict";
    }
}());
```